charts/flagsmith/templates/_api_environment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.databaseExternal.url }}
```

Comment:
I don't think this is right.

`DATABASE_URL` needs to be set, and `databaseExternal.url` is one of the ways it can get set, but not the only way.

Why is this change needed and how does it relate to the extra volumes and volume mounts?